### PR TITLE
feat: use the same std version across the repo

### PR DESCRIPTION
**Pull request by alexewerlof**
Use the deno.json `imports` convention to use the same std version throughout the repo

### CODE REVIEW

**Comment by userpixel** [comment]
While we're at it, we could upgrade to the latest deno std version which [is](https://deno.land/std) `0.204.0` at the time. I ran one of the examples and it worked. But since it's a bit outside the scope of this PR, I didn't do it.

File: deno.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+  "imports": {
+    "std/": "https://deno.land/std@0.204.0/"
```
Comment (alexewerlof) [inline deno.json:3]: I tested a couple of examples and they seem to be working fine

**Comment by lino-levan** [comment]
Unfortunately, this does not work. This would require every downstream consumer of of this module to have `std/` mapped to a version of the standard library. I'd be willing to accept this PR if it just updated the standard library version though!

**Comment by alexewerlof** [comment]
@lino-levan are you fine with the [deps.ts](https://docs.deno.com/runtime/tutorials/manage_dependencies) convention?

**Comment by lino-levan** [comment]
I don't mind. Whatever you think is best.

**Comment by alexewerlof** [comment]
#36 deprecates this one.

**Pull request closed without merging**